Gemfile

```diff
@@ -24,6 +24,8 @@ gem 'jbuilder', '~> 2.0'
 # bundle exec rake doc:rails generates the API under doc/api.
 gem 'sdoc', '~> 0.4.0', group: :doc
 
+gem 'smarter_csv', require: false
```

Comment:
Any reason to use smarter_csv? Seems that what you're doing is simple enough that you could just use the standard library CSV.